chia/util/block_tools.py

```diff
@@ -87,8 +87,8 @@
         "MAX_FUTURE_TIME": 3600
         * 24
         * 10,  # Allows creating blockchains with timestamps up to 10 days in the future, for testing
+        "COST_PER_BYTE": 1337,
```

Comment:
These are just for tests, right? this isn't supposed to mirror the default constants I take it

Reply:
yes it overrides the defaults